ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ptp.schema.yml

```diff
@@ -14,6 +14,8 @@ keys:
         valid_values:
         - boundary
         - transparent
+      boundary_one_step:
```

Comment:
An alternative implementation could be to add "boundary one-step" to valid values above.

Reply:
nevermind.